.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
 install:
   - npm install
 before_script:
-  - npm install -g @angular/cli codecov
+  - npm install -g codecov
```

Comment:
Installing the global cli give you the commands on the path, do we want to remove that?